backstory/src/main/java/com/google/sps/servlets/data/ImagesManagerFactory.java

```diff
@@ -0,0 +1,6 @@
+import com.google.sps.images.ImagesManager;
+
+/**
```

Comment:
Nit: All of these factory interfaces have the same class description, it wouldn't be a bad thing to at least specify the class it's for (even though it's somewhat obvious).